projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -2064,6 +2064,10 @@
     %igx-grid__loading-outlet {
         z-index: 10003;
 
+        > .igx-overlay__wrapper--modal {
```

Comment:
I can't have this like that. It should either reference a placeholder selector or a tag selector.